Dockerfile.QA

```diff
@@ -242,6 +243,10 @@ RUN cp client/src/grpc_generated/go/*.go qa/L0_simple_go_client/. && \
     cp client/src/grpc_generated/javascript/*.js qa/L0_simple_nodejs_client/. && \
     cp client/src/grpc_generated/javascript/*.json qa/L0_simple_nodejs_client/. && \
     cp -r client/src/grpc_generated/java qa/L0_client_java/.
+# Copy Java API jar
+RUN if [ "$TARGETPLATFORM" != "linux/arm64" ]; then \
```

Comment:
I didn't realize this won't work on arm64. Will this work on every other platform we have? (Windows, Jetpack, etc.)

Reply:
Currently no...

Reply:
Would it make sense to make the logic inclusive rather than exclusive here then? E.g. if $TARGETPLATFORM is in <list of supported platforms>; then...

Reply:
Agreed. See what I did in https://github.com/triton-inference-server/server/blob/48cb049883783b7860f6e6e7cc9fc39ce24457b5/Dockerfile.sdk

Maybe we want to add a TRITON_ENABLE_JAVA ARG?

Reply:
I would rather to be dependent on platforms since I think adding so many variables introduce more clutter. Is this ok?